controllers/conditions.go

```diff
@@ -158,6 +181,7 @@ func (r *ImageClusterInstallReconciler) setClusterInstalledConditions(ctx contex
 		Message: v1alpha1.InstallSucceededMessage,
 	})
 
+	r.Log.Info("Setting cluster installed conditions")
```

Comment:
Shouldn't we check the return of these `setClusterInstallCondition` calls?

Reply:
not in this case as we will set this condition once as reconcile will exit in case it was set already

Reply:
And yet, let's check and not relay on some other code to returnearly, this is exactly what this pr should do.

Reply:
why? there is not issue with it? We should not overcomplicate each part of the the code, there is a unitest that check that we don't run after setting installed